Message-Id: <d7d5684690a5110e57b4f5a5ba48b76aff0f183c.1336068799.git.agx@sigxcpu.org>
In-Reply-To: <87e0b605aa093d95c507083de601261098cd1aff.1336068799.git.agx@sigxcpu.org>
References: <5909887366833c92178e3ed650b87d666f9f50c1.1336068799.git.agx@sigxcpu.org>
	<56db9d711b7615ece3848c57f71f362f0e5f4fea.1336068799.git.agx@sigxcpu.org>
	<727a1c60fdc68e24b2f457f5b7e3611b2aae7a7b.1336068799.git.agx@sigxcpu.org>
	<af426bb880140c2941366a3a84fd11a714b9c778.1336068799.git.agx@sigxcpu.org>
	<64360f6da8f2d82618e0eda5d971ee6a083851ae.1336068799.git.agx@sigxcpu.org>
	<308046ae1d09cc7cce80fdb268cfb7fee35de13d.1336068799.git.agx@sigxcpu.org>
	<f96b7f473f61a8e6d4866a49d9fcc861bed443b5.1336068799.git.agx@sigxcpu.org>
	<7e773c9b5d95a42321be6b5fc7a11cf0fd51a124.1336068799.git.agx@sigxcpu.org>
	<6d5763889d5f114ed6f149ecd47c562f06868519.1336068799.git.agx@sigxcpu.org>
	<e82f2035cd9341f3530199e68c11a27790f041de.1336068799.git.agx@sigxcpu.org>
	<87e0b605aa093d95c507083de601261098cd1aff.1336068799.git.agx@sigxcpu.org>
From: =?UTF-8?q?Guido=20G=C3=BCnther?= <agx@sigxcpu.org>
Date: Sat, 2 Apr 2011 10:37:16 +0200
Subject: Brand ja

---
 ja/locale/ja/calendar/migration.properties  |    2 +-
 ja/locale/ja/calendar/timezones.properties  |    2 +-
 ja/locale/ja/lightning/lightning.dtd        |    2 +-
 ja/locale/ja/lightning/lightning.properties |    6 +++---
 4 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/ja/locale/ja/calendar/migration.properties b/ja/locale/ja/calendar/migration.properties
index 0b571e4..8f793c0 100644
--- a/ja/locale/ja/calendar/migration.properties
+++ b/ja/locale/ja/calendar/migration.properties
@@ -43,4 +43,4 @@ migrationTitle = %1$S: データのインポート
 migrationDescription=%1$S は他のプログラムからデータをインポートできます。以下のプログラムのデータが見つかりました。どのデータをインポートするか選択してください。
 finished = 完了
 disableExtTitle = 互換性のない拡張が見つかりました
-disableExtText = Lightning と互換性のない、古い Mozilla Calendar 拡張機能がインストールされています。この拡張機能を無効にして %1$S を再起動します。
+disableExtText = Iceowl と互換性のない、古い Mozilla Calendar 拡張機能がインストールされています。この拡張機能を無効にして %1$S を再起動します。
diff --git a/ja/locale/ja/calendar/timezones.properties b/ja/locale/ja/calendar/timezones.properties
index 3c61c5e..6578404 100644
--- a/ja/locale/ja/calendar/timezones.properties
+++ b/ja/locale/ja/calendar/timezones.properties
@@ -41,7 +41,7 @@
 
 # extension:
 extensions.calendar-timezones@mozilla.org.name=Timezone Definitions for Mozilla Calendar
-extensions.calendar-timezones@mozilla.org.description=Sunbird と Lightning 用のタイムゾーンデータ
+extensions.calendar-timezones@mozilla.org.description=Iceowl と Iceowl 用のタイムゾーンデータ
 extensions.calendar-timezones@mozilla.org.creator=Mozilla Calendar Project
 
 pref.timezone.floating=現地時間/タイムゾーンなし (Local Time/floating)
diff --git a/ja/locale/ja/lightning/lightning.dtd b/ja/locale/ja/lightning/lightning.dtd
index 696c53f..cfd4c77 100644
--- a/ja/locale/ja/lightning/lightning.dtd
+++ b/ja/locale/ja/lightning/lightning.dtd
@@ -46,7 +46,7 @@
   -->
 
 <!-- Tools menu -->
-<!ENTITY lightning.taskLabel				"Lightning">
+<!ENTITY lightning.taskLabel				"Iceowl">
 
 <!-- New menu popup in File menu -->
 <!ENTITY lightning.menupopup.new.event.label		"予定...">
diff --git a/ja/locale/ja/lightning/lightning.properties b/ja/locale/ja/lightning/lightning.properties
index de1eb90..99a1a72 100644
--- a/ja/locale/ja/lightning/lightning.properties
+++ b/ja/locale/ja/lightning/lightning.properties
@@ -38,12 +38,12 @@
 # ***** END LICENSE BLOCK *****
 
 # addon description:
-extensions.{e2fda1a4-762b-4020-b5ad-a41df1933103}.name=Lightning
+extensions.{e2fda1a4-762b-4020-b5ad-a41df1933103}.name=Iceowl
 extensions.{e2fda1a4-762b-4020-b5ad-a41df1933103}.description=Thunderbird に統合されたカレンダー
 extensions.{e2fda1a4-762b-4020-b5ad-a41df1933103}.creator=Mozilla Calendar Project
 
 # Lightning branding
-brandShortName=Lightning
+brandShortName=Iceowl
 
 # Task mode title
 taskModeApplicationTitle=ToDo
@@ -72,7 +72,7 @@ imipBarRequestText=メッセージに参加依頼が含まれています。
 imipBarUpdateText=メッセージに既存の予定の更新が含まれています。
 imipBarAlreadyProcessedText=メッセージにすでに終了した予定が含まれています。
 imipBarReplyText=メッセージに参加依頼への返信が含まれています。
-imipBarUnsupportedText=メッセージにこのバージョンの Lightning では対応できない予定が含まれています。
+imipBarUnsupportedText=メッセージにこのバージョンの Iceowl では対応できない予定が含まれています。
 imipBarProcessingFailed=メッセージの処理に失敗しました。状態: %1$S
 imipAcceptInvitation.label=承諾
 imipCancelInvitation.label=削除
